test_scripts/API/Restructuring_OnResetTimeout/005_BC_OnResetTimeout_sends_twice_notification.lua

```diff
@@ -0,0 +1,66 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check SDL is able to respond with GENERIC_ERROR:false to Mobile app in case:
+--  - two reset periods received withing 2 'OnResetTimeout(resetPeriod)' notifications from HMI are expired
+--  - and HMI hasn't responded
+-- Applicable RPCs: 'SendLocation', 'Alert', 'SubtleAlert', 'PerformInteraction', 'Slider', 'Speak',
+--  'ScrollableMessage', 'DiagnosticMessage', 'SetInteriorVehicleData'
+------------------------------------------------------------------------------------------------------------------------
+-- Preconditions:
+-- 1) Default SDL timeout is 10s (defined in .INI by 'DefaultTimeout' parameter)
+--
+-- In case:
+-- 1) App sends applicable RPC
+-- 2) SDL transfers this request to HMI
+-- 3) HMI sends 1st 'BC.OnResetTimeout' notification to SDL right after receiving request with 'resetPeriod=15s'
+-- 4) HMI sends 2nd 'BC.OnResetTimeout' notification to SDL after delay of 12s with 'resetPeriod=7s'
+-- 5) HMI doesn't provide a response
+-- SDL does:
+--  - wait for the response from HMI within 'delay + 2nd resetPeriod' (19s)
+--  - respond with GENERIC_ERROR:false to Mobile app once this timeout expires
+------------------------------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Local Variables ]]
+local paramsForRespFunctionFirstNot = {
+  notificationTime = 0,
+  resetPeriod = 15000
+}
+
+local paramsForRespFunctionSecondNot = {
+  notificationTime = 12000,
+  resetPeriod = 7000
+}
+
+--[[ Local Functions ]]
+local function diagnosticMessageError()
+  local requestParams = { targetID = 1, messageLength = 1, messageData = { 1 } }
+  local cid = common.getMobileSession():SendRPC("DiagnosticMessage", requestParams)
+
+  EXPECT_HMICALL("VehicleInfo.DiagnosticMessage", requestParams)
```

Comment:
I would propose to replace EXPECT_HMICALL(...) with common.getHMIConnection():ExpectRequest(...)
```suggestion
  common.getHMIConnection():ExpectRequest("VehicleInfo.DiagnosticMessage", requestParams)
```
This comment relates to other scripts of this PR.